outputs.tf

```diff
@@ -165,3 +165,19 @@ output "node_groups_iam_role_arns" {
     node_group.node_group_name => node_group.node_role_arn
   }
 }
+
+output "node_groups" {
+  value = [
```

Comment:
could you please add description ?

Reply:
Sure. Was planning on adding a docs `.md` describing matters in more detail. First I wanted to see if there's feedback on the output structure.